contracts/cw-dao/src/contract.rs

```diff
@@ -352,6 +382,29 @@ pub fn execute_update_config(
         .add_attribute("sender", info.sender))
 }
 
+pub fn execute_update_cw20_token_list(
+    deps: DepsMut,
+    env: Env,
+    info: MessageInfo,
+    mut to_add: Vec<Addr>,
+    to_remove: Vec<Addr>,
+) -> Result<Response<Empty>, ContractError> {
+    // Only contract can call this method
+    if env.contract.address != info.sender {
```

Comment:
fine for now, but seems a little weird you have to use a proposal to do this 🤷‍♂️

Reply:
Maybe a bit heavy weight, but didn't think these methods should just be open to everyone. :shrug: